docs/development.md

```diff
@@ -121,7 +122,7 @@ The pieces that make up `support-frontend` are:
 * The CSS for a non-shareable component is located inside the `page.scss` file.  
  
 
-## Building process
+## CI build process
```

Comment:
Since you change the name, you have to update the link in line 12 to be #ci-build-process